R/utilities-plotting.R

```diff
@@ -289,23 +290,23 @@
 
   # X-axis label
   xLabel <- switch(plotType,
-    "TimeProfilePlotConfiguration" = ,
-    "ResVsTimePlotConfiguration" = xUnitString,
-    # Note that `yUnitString` here is deliberate.
-    #
-    # In case of an observed versus simulated plot, `yValues` are plotted on
-    # both x- and y-axes, and therefore the units strings are going to be the
-    # same for both axes.
-    "ObsVsPredPlotConfiguration" = paste0("Observed values (", yUnitString, ")"),
-    "ResVsPredPlotConfiguration" = paste0("Simulated values (", yUnitString, ")")
+                   "TimeProfilePlotConfiguration" = ,
```

Comment:
I am wondering where these changes in indentation come from? Does your `styler` do something different than mine?